v1.0/Specification.md

```diff
@@ -235,7 +235,7 @@ The following table gives guidelines for how to comply with the requirements in
 | --- | --- |
 | STATIC_1 | You must choose one of these file formats to preserve a static representation of the spreadsheet in: JPEG-2000, PDF/A, PNG or TIFF file formats. A wide range of other image file formats exists, but these are typically not used for preservation of data, unless the data was originally created in the file format.<br><br>This specification limits the number of possible file formats to use for preserving a static representation of the spreadsheet, because it is necessary to specify exact file formats, otherwise it is not possible to validate the file format. Simply stating that “your organisation must use any static file format, that your organisation has a file format policy for” is not sufficient, because it is not feasible for software validation workflows to universally validate any possible static file format. |
 | STATIC_2 | Extension should be lowercase. |
-| STATIC_3 | It is necessary to preserve each sheet in a spreadsheet, therefore you must make sure to export each sheet manually, or if applying a programmatic solution to loop all the sheets for exporting. It is up to your organisation to determine if sheets should be split into multiple export files, or if you combine them into a single multipage file. When using a single multipage file, each page must have the name of the sheet in it. This specification does not recommend one approach over the other, and if your organisation does not have any preference, you may allow both approaches. |
+| STATIC_3 | It is necessary to preserve each sheet in a spreadsheet, therefore you must make sure to export each sheet manually, or if applying a programmatic solution to loop all the sheets for exporting. It is up to your organisation to determine if sheets should be split into multiple export files, or if you combine them into a single multipage file. If using a single file for each sheet, the filename for each sheet must be the sheet's name. If using a single file for alle sheets, each page must have the name of the sheet in it. This specification does not recommend one approach over the other, and if your organisation does not have any preference, you may allow both approaches. |
 | STATIC_4 | Before exporting, you must applying text wrapping to all cells of the sheet, otherwise truncated cells with information will result in information loss, since this information will be hidden behind the proceeding column’s cell, if this cell contains information. |
```

Comment:
Similar again, this is getting to be a long piece of text that's falling over itself as it conflates a requirement and rationale. Guidelines might be better handled as text or have some accompanying context.

Reply:
I removed the part of multipage support. The requirement and guideline now mandates each sheet to be preserved as an indepdent file. This makes things much more clear and simple.

Reply:
I'm not sure that it does. Are we really dictating archival package structure for what's a dubious practice? The text about looping is unnecessary again and there's too much repetition of CSV-3.